test/mixer_music_test.py

```diff
@@ -178,6 +179,26 @@ def test_queue__invalid_filename(self):
         with self.assertRaises(pygame.error):
             pygame.mixer.music.queue("")
 
+    def test_music_pause__unpause(self):
+        """Ensure music has the correct position immediately after unpausing
+
+        |tags:music|
+        """
+        filename = example_path(os.path.join("data", "house_lo.mp3"))
+        pygame.mixer.music.load(filename)
+        pygame.mixer.music.play()
+
+        # Wait 0.5s, then pause
```

Comment:
Is it possible to change these sleeps to a lower value like maybe 0.05? So the tests run a bit quicker.

Reply:
Yes and done